.github/dependabot.yml

```diff
@@ -17,11 +17,11 @@ updates:
       interval: monthly
       day: monday
     reviewers:
-      - "mongodb/APIx1"
+      - "mongodb/APIx-Integrations"
```

Comment:
shouldn't CODEOWNERS catch this? perhaps we don't need it

Reply:
yeah I was tempted to remove it to be honest so I can do it